docs/manuals/source/DeveloperGuide/developerFaq.rst

```diff
@@ -0,0 +1,42 @@
+.. _developer-guide-faq:
+
+Developer FAQ
+=============
+
+Job run
+-------
+
+How to run a job that automatically fails?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+I want to run a job that automatically fails in order to test rescheduling on error.
```

Comment:
```suggestion
I want to run a job that automatically fails for testing purposes.
```
The question is too specific, keeping only the failing is enough in my opinion. The scheduling can be easily found using the documentation. The only new trick here is the `/bin/false`